### ISSUE TITLE
[Select][Web Components][2.22.0] Double Focus

### ISSUE THREAD

**Issue by kaloyantinchev** [UI5/webcomponents#13616]
### Bug Description

Currently, when activating the select drop-down, there are two visual focuses: one on the input, which also plays as the active state, and the second on a list item. This violates accessibility standards for focus, as there should only be one visible focus at any time. A secondary focus can cause confusion and may lead to additional accessibility issues.

### Affected Component

Select

### Expected Behaviour

When the drop-down is activated via the input button, the focus should automatically go to the selected item. The Focus/Active state on the dropdown should be removed. The only active state in the input should be the button while the drop-down is open.

<img width="490" height="323" alt="Image" src="https://github.com/user-attachments/assets/4553de29-a40f-42d7-af71-f3bcc0fa2f39" />

### Isolated Example

https://ui5.github.io/webcomponents/play/#eNq9U1FP2zAQ_ismT1DVNZuENIW0mgbbK0h0D5Py4trXxuDYVuy0VFX_-85OAkEMAS9TIiX3ne8733d3h0wZCY-zKtQ6yw-Z4TVk-RicZsKaACYgXJxQSpzm-01jWyPp2mpJKF2Upji5vrla_rn9SWJQBOKXaG428zIDU2aIxaeogEv8J_gWPuw1RGOSGxtOcwlrZUCekQNiUvmYKifGGrhE4BjjE3_yr20jQFJhtW0ol_etDy_OkoIN_P8_XVFD4ERUvPEQUIHfy1_0WxKhd0WhEd8q2DnbhDIjvcwI7pQM1VzCVgmgyZgSZVRQXFMvuIb5l9l5TxZUwJx3vHYaCtZZqDLrZS5WVu5JuhgSr7h46FuX6sjJljenlHrufjy5rqLn7JJUoDYV1vj14tw9pmz_aj8F049AV3erLqgHDSIgt25j1sBXGkJ_3-i3LihrnvwS_EOwDiVQqAECmrtkL647T8Geo94iGZIMHMpxiQzLBH-EwFXYy3F8Zy9u4_cVAXuuM5X-9mbg2IlGuUDC3sU8tZWtjol8I6LJlZnde0yE85MOpvbFtnV97NfpHemz43SgGq3xgIx3mE3GPJWSQCasNKqOY0jKbMZGbt-6CKcL4py_jk13eBH_HZVhO1gJi4jBnJ7hagV2l8QamN49fpO0_sBxGhvWB8Wmfzok9veTQd2AdkHZ8fgXUTfY9Q

### Steps to Reproduce

Press input button to activate select drop-down

### Log Output, Stack Trace or Screenshots

<img width="353" height="225" alt="Image" src="https://github.com/user-attachments/assets/db5df1d9-8f68-4d01-b746-d75a40503b1a" />

### Priority

None

### UI5 Web Components Version

2.22.0

### Browser

Chrome

### Operating System

macOS Tahoe 26.5

### Organization

Core Design

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by LidiyaGeorgieva** [comment]
Hello @UI5/ui5-team-ctr-pir  the issue is reproducible on the provided link. Pease have a look.
